tests/anomaly/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+set(hawktracer_anomaly_tests
+    main_tests.cpp
+    test_path.cpp
+    test_pattern_matching.cpp)
+
+file(COPY test1_config.json
```

Comment:
You might consider mixing `file(COPY)` with `file(GLOB)` so you don't have to list all the files explicitly